pagopa/pn-frontend
# fix/PN-9921: add target _blank to ToS and Privacy Link

## Short description
Add target _blank to ToS and Privacy Link

## List of changes proposed in this pull request
- add target _blank to redirectToSLink and redirectPrivacyLink in ToSAcceptance page per PF
- add target _blank to redirectToSLink and redirectPrivacyLink in ToSAcceptance page per PG
- add target _blank to redirectToSLink and redirectPrivacyLink in ToSAcceptance page per PA

## How to test
Login as PF/PG/PA and check links in ToSAcceptance redirect to a new tab.
NB: It could be necessary to set `response.data.accepted = false;` before returning response.data to see the page
